probe_scraper/probe_expiry_alert.py

```diff
@@ -112,10 +111,13 @@ def send_emails_for_expiring_probes(expired_probes, expiring_probes,
     logging.info(f"Sent emails to {len(probes_by_email_by_state)} recipients")
 
 
-def download_file(url, output_filepath):
-    content = requests.get(url).text
-    with open(output_filepath, "w") as output_file:
-        output_file.write(content)
+def get_latest_revision():
+    revision_info = requests.get(PROBE_INFO_BASE_URL + "firefox/revisions").json()["nightly"]
+    sorted_revisions = sorted(
+        [{'tag': tag, 'version': info['version']} for tag, info in revision_info.items()],
+        key=lambda x: x['version'], reverse=True
+    )
+    return sorted_revisions[0]['tag']
```

Comment:
Could use `max` here instead.
```suggestion
    return max(revision_info.items(), key=lambda x: x[1]['version'])[0]
```